client/analytics/components/report-summary/index.js

```diff
@@ -105,11 +114,31 @@ ReportSummary.propTypes = {
 		 */
 		key: PropTypes.string.isRequired,
 	} ).isRequired,
+	/**
+	 * Data to display in the SummaryNumbers.
+	 */
+	summaryData: PropTypes.object,
+};
+
+ReportSummary.defaultProps = {
+	summaryData: {
+		totals: {
+			primary: {},
+			secondary: {},
+		},
+		isError: false,
+		isRequesting: false,
+	},
 };
 
 export default compose(
 	withSelect( ( select, props ) => {
 		const { query, endpoint } = props;
+		if ( query.search && ! ( query[ endpoint ] && query[ endpoint ].length ) ) {
+			return {
+				emptySearch: true,
```

Comment:
Nitpicking here, but could we rename this to `emptyResults` or ``emptySearchResults`?  Someone else glancing at this might get confused and think this prop means no search query was made.

Reply:
Good point, makes a lot of sense!

Fixed in 21c1490d47028d06a15a101c401c89bb21fcdf52.

Reply:
Sweet- thanks!